Lib/pdb.py

```diff
@@ -1100,6 +1100,7 @@ def do_debug(self, arg):
             self.error(traceback.format_exception_only(*exc_info)[-1].strip())
             return
         p = Pdb(self.completekey, self.stdin, self.stdout)
+        p.use_rawinput = self.use_rawinput
```

Comment:
Could have a comment that this is required because "stdout" is passed.

As for a test, you could inspect frames (similar to https://github.com/antocuni/pdb/pull/169), but not sure if that is really useful / worth it.

(I just came across this again, after having backported it to pdbpp not correctly before - and wrote a test then.. ;))